app/src/containers/LoginScreen.tsx

```diff
@@ -80,28 +80,32 @@ class LoginScreen extends Container<ILoginScreenProps, ILoginScreenState> {
 				textType={"password"}
 			/>
 
-			<TextButton text={"Sign Up"} width={70} onAction={this.signUpButtonOnClick}/>
+			<TextButton buttonColour={"#6236ff"} buttonHoverColour={"#6236ff"} text={"Sign Up"} width={80} onAction={this.signUpButtonOnClick}/>
 		</div>;
 
 		return (
 			<div
 				className={"fill-container center-mid"}
 			>
 				<div style={{
-					marginBottom: 180,
+					marginBottom: 15,
+					marginRight: 100
 				}}>
-					<Image path={"/logo.png"} width={600} height={140}/>
+					<img src={"/musico.svg"} alt="Logo" width={800} height={500} />
```

Comment:
why are you using `<img/>` instead of `<Image/>` ?

Reply:
Idk how to change the “object-fit” from “cover” to “contain” using <Image />